openedx/core/djangoapps/site_configuration/models.py

```diff
@@ -263,11 +263,15 @@ def compile_microsite_sass(self):
                 'sass_compile_message': developer_message,
             }
 
+        if self.api_adapter:
```

Comment:
Matches [the theme code](https://github.com/appsembler/edx-theme-customers/blob/b7c37923620174c71b040d04ef5d82818dd6d0d8/lms/templates/theme-variables.html#L40-L41):

```suggestion
        theme_version = self.get_value('THEME_VERSION', 'amc-v1')
        if theme_version == 'tahoe-v2':
            # Use the Dashboard 2.0 variable names
```